src/component/category-form/category-form.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import autoBind from '../../utils/autobind';
+
+const defaultState = {
+  name: '',
+};
```

Comment:
I get a very loud warning from React about `controlled` vs `uncontrolled` components.
To get rid of that error, please add `budget: 0` into your default state. Read the docs here about controlled vs uncontrolled components.
https://reactjs.org/docs/forms.html#controlled-components